pulsar-broker/src/main/java/org/apache/pulsar/client/impl/RawBatchConverter.java

```diff
@@ -139,8 +142,22 @@ public static Optional<RawMessage> rebatchMessage(RawMessage msg,
 
                 ByteBuf metadataAndPayload = Commands.serializeMetadataAndPayload(Commands.ChecksumType.Crc32c,
                                                                                   metadata, compressedPayload);
-                Optional<RawMessage> result = Optional.of(new RawMessageImpl(msg.getMessageIdData(),
-                                                                             metadataAndPayload));
+
+                if (brokerEntryMetadata != null) {
+                    int brokerMetaSize = brokerEntryMetadata.getSerializedSize();
+                    ByteBuf brokerMeta =
+                            PulsarByteBufAllocator.DEFAULT.buffer(brokerMetaSize + 6, brokerMetaSize + 6);
```

Comment:
Please help give a static name for the number 6. it's hard for developers to know what it is.